src/analyzer.rs

```diff
@@ -475,12 +469,19 @@ fn extract_integer_type(value: &JSONSchemaProps) -> Result<String> {
     })
 }
 
-fn uppercase_first_letter(s: &str) -> String {
-    let mut c = s.chars();
-    match c.next() {
-        None => String::new(),
-        Some(f) => f.to_uppercase().collect::<String>() + c.as_str(),
-    }
+fn to_camel_case(s: &str) -> String {
```

Comment:
we already have `heck` in the dependency graph
possibly worth using https://docs.rs/heck/latest/heck/trait.ToUpperCamelCase.html here and see if the tests hold

Reply:
I'll change it, looking through their testcases it does what we need